src/main/webapp/app/overview/course-exercises/course-exercises.component.ts

```diff
@@ -60,15 +84,68 @@ export class CourseExercisesComponent implements OnInit, OnDestroy {
         this.paramSubscription.unsubscribe();
     }
 
-    public groupExercises(selectedOrder: number): void {
+    get filterType() {
+        return ExerciseFilter;
+    }
+
+    /**
+     * Return the total number of exercises for the logged in user
+     */
+    getNrOfExercises(): number {
+        return sum([...this.exerciseCountMap.values()]);
```

Comment:
Not part of the scope of this PR as introduced in another PR, but the use of Sets/Maps really makes the control flow a lot harder...

If `this.exerCountMap = {}` this would be: `Object.values(this.exerciseCountMap).reduce(sum, 0)`;